README.md

```diff
@@ -214,6 +214,9 @@ degree documented below):
   supported on Windows. We also test `i686-pc-windows-msvc`, with the same
   reduced feature set. We might ship Miri with a nightly even when some features
   on these targets regress.
+- `aarch64-linux-android` has basic support and should work for simple targets.
```

Comment:
What are "simple targets"?

Also it seems a bit premature to add this if we don't have anything on CI and people need to patch stdlib to make it work.